checker/checkLibrary.ml

```diff
@@ -335,7 +335,7 @@ let marshal_in_segment (type a) ~validate ~value ~(segment : a ObjFile.segment)
 
 let summary_seg : summary_disk ObjFile.id = ObjFile.make_id "summary"
 let library_seg : library_disk ObjFile.id = ObjFile.make_id "library"
-let opaques_seg : seg_proofs ObjFile.id = ObjFile.make_id "opaques"
+let opaques_seg : seg_proofs ObjFile.id = ObjFile.make_id "opaque proofs"
```

Comment:
This change and the one below seem pretty much unrelated to this PR. Also I think that adding spaces in the segment name looks like a bad idea, couldn't we just keep the previous name?

Reply:
The segment name gets printed in the "loading ... from disk" message in this PR, so this change avoids needing an overlay for dpdgraph.

Reply:
I'd still rather have an overlay than adding a space in there. These things should be thought of as simple identifiers into the file, not something the upper layers should rely on to print stuff to the user.

Reply:
changed